__tests__/app.test.js

```diff
@@ -32,3 +33,38 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  it("returns a status 200 and an array of articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toBeInstanceOf(Array);
+      });
+  });
+  it("returns objects from the articles table with the correct keys ", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).not.toBe(0);
+        expect(body.articles[0]).toHaveProperty("author");
+        expect(body.articles[0]).toHaveProperty("title");
+        expect(body.articles[0]).toHaveProperty("article_id");
+        expect(body.articles[0]).toHaveProperty("topic");
+        expect(body.articles[0]).toHaveProperty("created_at");
+        expect(body.articles[0]).toHaveProperty("votes");
+        expect(body.articles[0]).toHaveProperty("article_img_url");
+        expect(body.articles[0]).toHaveProperty("comment_count");
+      });
```

Comment:
This test is simply asserting that the first article in the articles array has these properties, i would recommend itterating over this array and asserting that every article is what you expect it to be. 

I would also recommend asserting that they are a specific data type instead of simply checking if they are properties that exist on each article.

Reply:
added to test suite